lua/customroles/soulbound.lua

```diff
@@ -0,0 +1,611 @@
+            local id = client:GetNWString("TTTSoulboundAbility" .. slot, "")
+            local ability = SOULBOUND.Abilities[id]
+            if #id == 0 or not ability then
+                y = y - titleHeight - margin
+                draw.RoundedBox(8, x, y, width, titleHeight, Color(20, 20, 20, 200))
+                draw.RoundedBoxEx(8, x, y, titleHeight, titleHeight, Color(90, 90, 90, 255), true, false, true, false)
+                draw.SimpleText("Unassigned", "TimeLeft", x + titleHeight + (margin * 2), y + (titleHeight / 2), COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_CENTER)
+            else
+                y = y - titleHeight - bodyHeight - (margin * 2)
+                draw.RoundedBox(8, x, y, width, titleHeight + bodyHeight + margin, Color(20, 20, 20, 200))
+                draw.SimpleText(ability.Name, "TimeLeft", x + titleHeight + (margin * 2), y + (titleHeight / 2), COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_CENTER)
+                local ready = ability:DrawHUD(client, x, y + titleHeight + margin, width, bodyHeight, Key("slot" .. slot, slot))
+                local slotColor = Color(90, 90, 90, 255)
+                if ready then
+                    slotColor = ROLE_COLORS[ROLE_SOULBOUND]
+                end
+                draw.RoundedBoxEx(8, x, y, titleHeight, titleHeight, slotColor, true, false, false, true)
+            end
+            CRHUD:ShadowedText(slot, "Trebuchet22", x + (titleHeight / 2), y + (titleHeight / 2), COLOR_WHITE, TEXT_ALIGN_CENTER, TEXT_ALIGN_CENTER)
+        end
+    end)
+
+    hook.Add("HUDDrawScoreBoard", "Soulbound_HUDDrawScoreBoard", function() -- Use HUDDrawScoreBoard instead of HUDPaint so it draws above the TTT HUD
+        if GetRoundState() ~= ROUND_ACTIVE then return end
+
+        if not client then
+            client = LocalPlayer()
+        end
+        if not client:IsSoulbound() then return end
+
+        local margin = 10
+        local height = 32
+        draw.RoundedBox(8, margin, ScrH() - height - margin, 170, height, ROLE_COLORS[ROLE_SOULBOUND])
+        if #ROLE_STRINGS[ROLE_SOULBOUND] > 10 then
+            CRHUD:ShadowedText(ROLE_STRINGS[ROLE_SOULBOUND], "TraitorStateSmall", margin + 84, ScrH() - height - margin + 2, COLOR_WHITE, TEXT_ALIGN_CENTER)
+        else
+            CRHUD:ShadowedText(ROLE_STRINGS[ROLE_SOULBOUND], "TraitorState", margin + 84, ScrH() - height - margin, COLOR_WHITE, TEXT_ALIGN_CENTER)
+        end
+    end)
+
+    --------------
+    -- TUTORIAL --
+    --------------
+
+    hook.Add("TTTTutorialRoleText", "Soulbound_TTTTutorialRoleText", function(role, titleLabel)
+        if role == ROLE_SOULBOUND then
+            local roleColor = ROLE_COLORS[ROLE_TRAITOR]
+
+            local html = "The " .. ROLE_STRINGS[ROLE_SOULBOUND] .. " is a member of the <span style='color: rgb(" .. roleColor.r .. ", " .. roleColor.g .. ", " .. roleColor.b .. ")'>traitor team</span> who can use special powers while dead to help the traitor team."
```

Comment:
Reads slightly better
```suggestion
            local html = "The " .. ROLE_STRINGS[ROLE_SOULBOUND] .. " is a member of the <span style='color: rgb(" .. roleColor.r .. ", " .. roleColor.g .. ", " .. roleColor.b .. ")'>traitor team</span> who can use special powers while dead to help their fellow traitors."
```